src/github.com/getlantern/autoupdate-server/server/asset_test.go

```diff
@@ -0,0 +1,16 @@
+package server
+
+import (
+	"fmt"
+	"testing"
+)
+
+const (
+	testAssetURL = `https://github.com/getlantern/autoupdate-server/releases/download/0.4.0/autoupdate-binary-darwin-amd64.v4`
+)
+
+func TestDownloadAsset(t *testing.T) {
+	if _, err := downloadAsset(testAssetURL); err != nil {
+		t.Fatal(fmt.Errorf("Failed to download asset: %q", err))
+	}
+}
```

Comment:
Right. I'm suggesting as a general approach it might make more sense to always test everything in org/getlantern or to use a blacklist. That way we don't have to remember to add something to testpackages.txt every time we add a new package.